src/Php81/Resources/stubs/ReturnTypeWillChange.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+#[Attribute]
```

Comment:
doesn't it have any flag? eg TARGET_METHOD?

Reply:
/cc @kocsismate should we add it to php-src?

Reply:
I actually doesn't, see my dumper output above.

Reply:
We should do the same than php-src. But it might make sense for php-src to configure the target there.

Reply:
Yes, not having a target on this attribute looks like a mistake.

Reply:
https://github.com/php/php-src/pull/6971/files#diff-7fa10e4571ac44e902ec9885a4854f676b2e7f5ccc21711a2b626ac9d6e3f289R289 makes me think that the attribute is actually already restricted to method targets, but that this is not properly exposed to Reflection
/cc @kocsismate @nikic

Reply:
Thanks for the report, I'll look into it.

Reply:
Should be fixed by https://github.com/php/php-src/commit/db6e60e744ab0d98d10b74cff3e897d5ff2f7574.

Reply:
I can confirm that reflection now reports `4` as argument which is the value of `Attribute::TARGET_METHOD`.

Reply:
I've updated the screenshot.